extension/js/common/api/key-server/attester.ts

```diff
@@ -25,7 +25,34 @@ export class Attester extends Api {
       console.info(`Skipping attester lookup of ${email} because attester search on this domain is disabled.`);
       return { pubkeys: [] };
     }
-    return await this.doLookup(email);
+    // Can't use Promise.allSettled because we are using ES2018 (Available from ES2020)
```

Comment:
You can set `minimum_chrome_version` in manifest to `96`, and start using `ES2020`. It's about time :-) then you won't need this shim.